admin/data/sql/migrations/026.do.create-pupil-restart-codes.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [mtc_admin].[pupilRestartCode] (
```

Comment:
This will need a rename now.. I think the next sequence is 28.

Reply:
@jon-shipley incremented